libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -426,8 +427,9 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     this.changeActiveView(index, TimeUnit.months);
   }
 
-  public changeYear(year: number) {
+  _changeYear(year: number) {
     this.changeActiveView(year - getYear(this.activeMonth), TimeUnit.years);
+    this.yearChange.emit(+year);
```

Comment:
```suggestion
    this.yearSelect.emit(+year);
```

Reply:
Just out of curiosity, what does the `+` in `+year` do?

Reply:
It parses the year from a string to a number. The parameter is number, but `year: number` is a string.

If you try this in `calendar-card-example.component.ts` it will be false and remove the '+' from '+year'.
`testYearSelect(year) {  console.log(year);  console.log(typeof year === 'number'); }`

![image](https://user-images.githubusercontent.com/77673731/126501138-aceea5fb-5ac4-4e90-9aa6-a13b07e91702.png)

Reply:
Ah I see makes sense - now that we have the file open I suppose we should correct the type annotation with something like this: 

```typescript
_changeYear(year: string) {
    const yearNumeric = Number(year);
    this.changeActiveView(yearNumeric - getYear(this.activeMonth), TimeUnit.years);
    this.yearSelect.emit(yearNumeric);
  }
```

I personally prefer to cast to number using `Number(value)` as it is more explicit what is going on 😄

Reply:
Yes ofcause. This is changed